src/constants/types.rs

```diff
@@ -125,10 +125,11 @@ js_deserializable!(StructureType);
 ///
```

Comment:
If it wouldn't be a nuisance, could you update the doc-comment for `IntershardResourceType` to mention `__resource_type_str_to_num` rather than `__intershard_resource_type_str_to_num` to accompany the change?